### DISCUSSION TITLE
How to exclude an term from the alert email

### DISCUSSION THREAD

**Discussion by rshiva777** [jertel/elastalert2#1063] [Q&A] [answered]
I have enabled email alerting for elasticsearch, and i am able to receive alerts via email. 
A sample email alert as below
```
At least 20 events occurred between 2022-12-24 03:04 UTC and 2022-12-24 03:05 UTC

@timestamp: 2022-12-24T03:05:46.112Z
_id: 6b8XQoUBuPbmf-NI_En8
_index: logstash-2022.12.24
kubernetes: {
    "annotations": {
        "cni_projectcalico_org/podIP": "192.168.1.2/32",
        "cni_projectcalico_org/podIPs": "192.168.1.2/32"
    },
    "container_hash": "[test.registry.com)",
    "container_image": "test.registry.com/nginx:1.20.1",
    "container_name": "nginx",
    "docker_id": "4011de151494e0e42127d8ecd0420ee306c66ee594d996a6a5f19fb0f4318c74",
    "host": "master-01",
    "labels": {
        "app": "nginx02",
        "pod-template-hash": "6fc5d69ffd"
    },
    "namespace_name": "default",
    "pod_id": "b1651ff6-6a89-43ca-9671-9e8c220621e6",
    "pod_name": "nginx02-6fc5d69ffd-6j94c"
}
logtag: F
message: shivakum
num_hits: 69
num_matches: 3
stream: stdout

```

The thing is, i dont want to get all of the above filed in the alert. i need only some information 
i want to exclude   the below fileds. Could anyone please let me know how to achive this
**kubernetes.annotations, container_hash, image, docker_id**

**Comment by jertel** [comment, marked as answer]
There are a lot of options for formatting your alerts. Read through this portion of the docs to get started: https://elastalert2.readthedocs.io/en/latest/ruletypes.html#alert-subject

**Reply by rshiva777** [reply]
@jertel, Thanks for the response, i have gone through the documentation but unable to get proper idea to do so. Could you please give an example for it


At least 20 events occurred between 2022-12-24 03:04 UTC and 2022-12-24 03:05 UTC

```
@timestamp: 2022-12-24T03:05:46.112Z
_id: 6b8XQoUBuPbmf-NI_En8
_index: logstash-2022.12.24
kubernetes: {
    "annotations": {
        "cni_projectcalico_org/podIP": "192.168.1.2/32",
        "cni_projectcalico_org/podIPs": "192.168.1.2/32"
    },
    "container_hash": "[test.registry.com)",
    "container_image": "test.registry.com/nginx:1.20.1",
    "container_name": "nginx",
    "docker_id": "4011de151494e0e42127d8ecd0420ee306c66ee594d996a6a5f19fb0f4318c74",
    "host": "master-01",
    "labels": {
        "app": "nginx02",
        "pod-template-hash": "6fc5d69ffd"
    },
    "namespace_name": "default",
    "pod_id": "b1651ff6-6a89-43ca-9671-9e8c220621e6",
    "pod_name": "nginx02-6fc5d69ffd-6j94c"
}
logtag: F
message: shivakum
num_hits: 69
num_matches: 3
stream: stdout
```

i want to exclude **kubernetes.annotations** from the email alert

**Reply by jertel** [reply]
Please paste the rule YAML showing what you've tried, specifically the alert_text_args parameter, and related, and we'll try to help you locate the problem. Include any error messages that you are seeing.
